src/sensors/scripts/lidar.py

```diff
@@ -11,13 +12,31 @@ class Lidar:
     """
 
     def __init__(self):
-        self.lidar_pub = rospy.Publisher('lidar',PointCloud ,queue_size=10)
+        self.lidar_pub = rospy.Publisher('lidar', PointCloud ,queue_size=10)
 
     def send_lidar_data(self):
         rospy.init_node('talker', anonymous=True)
-        rate = rospy.Rate(2)
+
+        client = airsim.CarClient()
+        client.confirmConnection()
+
+        rate = rospy.Rate(30)
```

Comment:
Like with navigation, is 30 too high or is it fine. I doubt there is any good way of measuring this, but do you think decreasing it would affect performance?

Reply:
I'm not quite sure honestly, need to play around. I think that is more of a minor point right now though